server/pubspec.yaml

```
@@ -24,7 +24,7 @@ dependencies:
   a_bridge: ^0.0.2
   device_info_plus: any
   image_picker: ^1.1.2
-  path: 1.9.0
+  path: 1.9.1
```

I would suggest to have a caret syntax here for better compatibility.

```suggestion
  path: ^1.9.0
```